spec/system/manage_users/deactivate_a_user_spec.rb

```diff
@@ -98,9 +98,10 @@ def do_deactivate_journey
   end
 
   describe 'with only 2 active admins' do
+    let(:user_can_manage_others) { true }
```

Comment:
I think this is already defaulted to true in the shared context no?

Reply:
One is the user, other is the current_user. I think name could be improved to avoid confusion.

Reply:
Yeah, I do find the naming with these users confusing. 

Gotcha, I was looking at with_an_existing_user in main and forgot that you'd added the can_manage_others_attribute 👍